Eatery Blue/UI/Onboarding/OnboardingFeaturesViewController.swift

```diff
@@ -158,10 +155,18 @@ class OnboardingFeaturesViewController: UIViewController {
         let clampedIndex = max(0, min(pages.count - 1, currentIndex))
         let nextIndex = clampedIndex + 1
 
+        // Mark : Changed nextButton text to "Lets go!" to transition onboarding to HomeModel view
+        
+        if nextIndex == pages.count - 1 {
+            nextButton.content.text = "Lets go!"
+        }
+
```

Comment:
This is a little picky, but you can probably combine these two if-statements into an "if" and "else if" block for better readability, since it can only be one situation or the other.